### PR TITLE
db schema migrations to match django db

### PR DESCRIPTION
**Pull request by eviljeff**
schematastic!
part of #10917 
split up into separate files as suggested by @EnTeQuAk in other pr

### CODE REVIEW

File: src/olympia/migrations/1132-db-schema-addons.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    ADD CONSTRAINT `addons_description_2300852e_fk_translations_id` FOREIGN KEY (`description`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_summary_0d397f7c_fk_translations_id` FOREIGN KEY (`summary`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_developercomments_b365508d_fk_translations_id` FOREIGN KEY (`developercomments`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_eula_62f9d8e4_fk_translations_id` FOREIGN KEY (`eula`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_privacypolicy_5e0b364a_fk_translations_id` FOREIGN KEY (`privacypolicy`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_supporturl_1101e07c_fk_translations_id` FOREIGN KEY (`supporturl`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_supportemail_da41fe48_fk_translations_id` FOREIGN KEY (`supportemail`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_current_version_5a2c8cb3_fk_versions_id` FOREIGN KEY (`current_version`) REFERENCES `versions` (`id`),
+    ADD UNIQUE KEY `supportemail` (`supportemail`),
```
Comment (diox) [inline src/olympia/migrations/1132-db-schema-addons.sql]: On dev/stage, `supportemail` and `supporturl` already exist.

Comment (diox) [reply src/olympia/migrations/1132-db-schema-addons.sql]: Similarly, `supporturl`, which you haven't touched here, already exists but isn't `UNIQUE` either

File: src/olympia/migrations/1138-db-schema-reviews.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
+ALTER TABLE `reviews`
+    MODIFY `created` datetime(6) NOT NULL,
+    MODIFY `modified` datetime(6) NOT NULL,
+    MODIFY `id` int(10) unsigned NOT NULL,
```
Comment (diox) [inline src/olympia/migrations/1138-db-schema-reviews.sql]: I got `Cannot change column 'id': used in a foreign key constraint 'akismet_reports_rating_instance_id_d282058c_fk_reviews_id' of table 'olympia_dev_copy.akismet_reports` when running this

File: src/olympia/migrations/1145-db-schema-appversions.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
+ALTER TABLE `appversions`
+    MODIFY `id` int(10) unsigned NOT NULL,
```
Comment (diox) [inline src/olympia/migrations/1145-db-schema-appversions.sql]: Got `Cannot change column 'id': used in a foreign key constraint 'applications_versions_max_6e57db5a_fk_appversions_id' of table 'olympia_dev_copy.applications_versions`

File: src/olympia/migrations/1148-db-schema-cannedresponses.sql [hunk 4]
```diff
@@ -0,0 +1,9 @@
+ALTER TABLE `cannedresponses`
+    MODIFY `id` int(10) unsigned NOT NULL,
```
Comment (diox) [inline src/olympia/migrations/1148-db-schema-cannedresponses.sql]: `Cannot change column 'id': used in a foreign key constraint 'log_activity_comment_canned_response_id_6a9271d5_fk_cannedres' of table 'olympia_dev_copy.log_activity_comment_draft'`

File: src/olympia/migrations/1149-db-schema-categories.sql [hunk 5]
```diff
@@ -0,0 +1,10 @@
+ALTER TABLE `categories`
+    MODIFY `id` int(10) unsigned NOT NULL,
```
Comment (diox) [inline src/olympia/migrations/1149-db-schema-categories.sql]: `Cannot change column 'id': used in a foreign key constraint 'addons_categories_category_id_f4f5c093_fk_categories_id' of table 'olympia_dev_copy.addons_categories'`

File: src/olympia/migrations/1138-db-schema-reviews.sql (hunk 2 above)
Comment (eviljeff) [reply src/olympia/migrations/1138-db-schema-reviews.sql]: https://github.com/mozilla/addons-server/pull/12272/commits/a5ae7521db48294a7852c3bb971e907109fe4683

File: src/olympia/migrations/1132-db-schema-addons.sql (hunk 1 above)
Comment (eviljeff) [reply src/olympia/migrations/1132-db-schema-addons.sql]: https://github.com/mozilla/addons-server/pull/12272/commits/a5ae7521db48294a7852c3bb971e907109fe4683

**Pull request merged**